frontend/app/src/components/trade/send/TradeSendCard.vue

```diff
@@ -0,0 +1,432 @@
+<script setup lang="ts">
+import type { GasFeeEstimation } from '@/types/trade';
+import type BigNumber from 'bignumber.js';
+import AmountDisplay from '@/components/display/amount/AmountDisplay.vue';
+import TradeAmountInput from '@/components/trade/send/TradeAmountInput.vue';
+import TradeAssetSelector from '@/components/trade/send/TradeAssetSelector.vue';
+import TradeConnectedAddressBadge from '@/components/trade/send/TradeConnectedAddressBadge.vue';
+import TradeHistoryView from '@/components/trade/send/TradeHistoryView.vue';
+import TradeRecipientAddress from '@/components/trade/send/TradeRecipientAddress.vue';
+import { useTradeApi } from '@/composables/api/trade';
+import { useInterop } from '@/composables/electron-interop';
+import { useSupportedChains } from '@/composables/info/chains';
+import { useTradeAsset } from '@/composables/trade/asset';
+import { useWalletHelper } from '@/composables/trade/wallet-helper';
+import { useBlockchainAccountsStore } from '@/modules/accounts/use-blockchain-accounts-store';
+import { useAccountAddresses } from '@/modules/balances/blockchain/use-account-addresses';
+import { useBlockchainAccountData } from '@/modules/balances/blockchain/use-blockchain-account-data';
+import { useBalancesStore } from '@/modules/balances/use-balances-store';
+import { useTaskStore } from '@/store/tasks';
+import { useWalletStore } from '@/store/trade/wallet';
+import { CURRENCY_USD } from '@/types/currencies';
+import { TaskType } from '@/types/task-type';
+import { bigNumberSum } from '@/utils/calculation';
+import { logger } from '@/utils/logging';
+import { bigNumberify, Blockchain, isValidEthAddress, Zero } from '@rotki/common';
+
+const { t } = useI18n();
+
+const amount = ref<string>('');
+const asset = ref<string>('');
+const assetChain = ref<string>(Blockchain.ETH);
+const toAddress = ref<string>('');
+
+const max = ref<string>('0');
+const estimatingGas = ref<boolean>(false);
+const showNeverInteractedWarning = ref<boolean>(false);
+const errorMessage = ref<string>('');
+const currentGasEstimationController = ref<AbortController | null>(null);
```

Comment:
again here if we are the ones tracking it let's keep it simple and get rid of the null